homeassistant/components/saunum/config_flow.py

```diff
@@ -0,0 +1,83 @@
+"""Config flow for Saunum Leil Sauna Control Unit integration."""
+
+from __future__ import annotations
+
+import logging
+from typing import Any
+
+from pysaunum import DEFAULT_PORT, SaunumClient, SaunumException
+import voluptuous as vol
+
+from homeassistant.config_entries import ConfigFlow, ConfigFlowResult
+from homeassistant.const import CONF_HOST, CONF_PORT
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers import config_validation as cv
+
+from .const import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+STEP_USER_DATA_SCHEMA = vol.Schema(
+    {
+        vol.Required(CONF_HOST): cv.string,
+        vol.Required(CONF_PORT, default=DEFAULT_PORT): cv.port,
```

Comment:
the cv.string and cv.port can just be `str` and `int`

Reply:
cv.port also adds validation for range.
https://github.com/home-assistant/core/blob/ca5b9ce0d302290b82e6d01184233646c0fd3fd8/homeassistant/helpers/config_validation.py#L197